_includes/_markdown/install-munit.md

```diff
@@ -0,0 +1,42 @@
+{% altDetails install-info-box 'Getting MUnit' %}
+
+{% tabs munit-unit-test-1 class=tabs-build-tool %}
+{% tab 'Scala CLI' %}
+You can require the entire toolkit in a single line:
+```scala
+//> using toolkit "latest"
```

Comment:
One last thing - I think we should comment it out for the time being (until the toolkit-test feature is merged to the scala-cli and released, it should be a short time from now, it has a high priority). Instead, we should put there the direct dep to the toolkit-test artifact.